### PR TITLE
Makes the Clown All Powerful

### PR DESCRIPTION
**Pull request by Fox-McCloud**
It's time for the clown to ascend.

Gives the clown a sad trombone implant---when he dies,  it plays the sad trombone---what more could a clown want? Even in death, the honking must continue.

More generally, adds the sad trombone implant to the game; you can print one off from R&D if you have bananium and a handful of tech levels---or you might get lucky and find one on a dead clown in Lavaland.

:cl: Fox McCloud
add: Adds sad trombone implant; make it at R&D with a few tech levels and some bananium
add: Clown starts off with a sad trombone implant
/:cl:

### CODE REVIEW

**Comment by ppi13** [comment]
image

File: code/modules/research/designs/medical_designs.dm [hunk 1]
```diff
@@ -533,6 +533,16 @@
 	build_path = /obj/item/implantcase/chem
 	category = list("Medical")
 
+/datum/design/implant_sadtrombone
+	name = "Sad Trombone Implant Case"
+	desc = "Makes death amusing."
+	id = "implant_trombone"
+	req_tech = list("materials" = 3, "biotech" = 5,)
```
Comment (dearmochi) [inline code/modules/research/designs/medical_designs.dm:540]:
```suggestion
	req_tech = list("materials" = 3, "biotech" = 5)
```

**Review by dearmochi — APPROVED** [review]
_Womp womp womp womp_

File: code/modules/research/designs/medical_designs.dm (hunk 1 above)
Comment (Fox-McCloud) [reply code/modules/research/designs/medical_designs.dm:540]: *scream

File: code/game/objects/items/weapons/implants/implant_clown.dm [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/obj/item/implant/sad_trombone
+	name = "sad trombone implant"
+	activated = 0
```
Comment (ppi13) [inline code/game/objects/items/weapons/implants/implant_clown.dm:3]: Is it supposed to be 0 or false here?

**Comment by Qustinnus** [comment]
now port my BAZINGA  PR

**Comment by Fox-McCloud** [comment]
@Qustinnus 

Yeah, I wanted to do that, but we don't have the pie cannon or pie throwing, nor vendor crushing.

Though I suppose just having it apply to slips or when a clown shoots himself would be just fine too. Either case, I loved the feature.

**Comment by Aracino** [comment]
Even in death, his duty does not end.

**Review by Kyep — APPROVED** [review]

**Review by Dumbdumn5 — APPROVED** [review]
I need it

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**